src/Apps/Order/Routes/Shipping/FulfillmentDetailsForm.tsx

```diff
@@ -0,0 +1,830 @@
+                              errors.attributes?.phoneNumber
+                            }
+                            data-testid="AddressForm_phoneNumber"
+                          />
+                        </Column>
+                        <Spacer y={2} />
+                      </>
+                    }
+                  </GridColumns>
+                  <Spacer y={2} />
+
+                  <Checkbox
+                    tabIndex={tabbableFormValue("new_address")}
+                    onSelect={selected => {
+                      formikProps.setFieldValue(
+                        "attributes.saveAddress",
+                        selected
+                      )
+                    }}
+                    selected={values.attributes.saveAddress}
+                    data-testid="save-address-checkbox"
+                  >
+                    Save shipping address for later use
+                  </Checkbox>
+                  <Spacer y={4} />
+                </Collapse>
+              </>
+            )}
+            {/* PHONE NUMBER */}
+            <Collapse
+              data-testid="phoneNumberCollapse"
+              open={values.fulfillmentType === FulfillmentType.PICKUP}
+            >
+              {values.fulfillmentType === FulfillmentType.PICKUP && (
+                <>
+                  <Input
+                    tabIndex={tabbableFormValue("pickup")}
+                    id="attributes.name"
+                    placeholder="Full name"
+                    title={"Full name"}
+                    autoCorrect="off"
+                    value={values.attributes.name}
+                    onChange={handleChange}
+                    error={touched.attributes?.name && errors.attributes?.name}
+                    data-testid="AddressForm_name"
+                  />
+                  <Input
+                    tabIndex={tabbableFormValue("pickup")}
+                    name="attributes.phoneNumber"
+                    title="Phone number"
+                    type="tel"
+                    description={"Required for pickup logistics"}
+                    placeholder="Add phone"
+                    pattern="[^a-z]+"
+                    value={values.attributes.phoneNumber}
+                    onChange={handleChange}
+                    error={
+                      touched.attributes?.phoneNumber &&
+                      errors.attributes?.phoneNumber
+                    }
+                    data-testid="AddressForm_phoneNumber"
+                  />
+                  <Spacer y={4} />
+                </>
+              )}
+            </Collapse>
+            {/* <Button
+              type="submit"
+              loading={formikProps.isSubmitting || undefined}
+              variant="primaryBlack"
+              width="50%"
+              // disabled={formikProps.isValid}
+            >
+              Save and Continue (shipping)
+            </Button> */}
```

Comment:
Removed the local button and am passing the handler into a ref because in the current behavior the shipping form displays a continue button in two separate places depending on breakpoint. Personally I would advocate for treating these as 2 separate steps, with a unique button that appears below each form.